debian/rules

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/make -f
+
+VERSION=0.1
+DEB_DIR=debian/streamgraph$(VERSION)
+
+%:
+	dh $@
+
+override_dh_auto_clean:
+
+override_dh_auto_build:
+	dh_auto_build
+
+override_dh_auto_install:
+	mkdir -p $(DEB_DIR)/usr/bin
+	install -m 755 obj-x86_64-linux-gnu/bin/streamgraph $(DEB_DIR)/usr/bin/streamgraph$(VERSION)
```

Comment:
Is there a way to get an install file to apply a version suffix? Otherwise `cmake` would need to build the binaries with the version suffix already applied.